frontend/src/model.tsx

```diff
@@ -698,6 +698,24 @@ function SubmitJob(instanceId: any, user: any, job: any, successCallback?: Callb
   })
 }
 
+function SubmitJobScript(clusterName: string, instanceId: string, user: string, job: {[key: string]: string}, successCallback: Callback, failureCallback: Callback) {
+  const region = getState(['app', 'selectedRegion']) || getState(['aws', 'region']);
+  let url = `manager/submit_job_script?cluster_name=${clusterName}&instance_id=${instanceId}&user=${user || 'ec2-user'}&region=${region}`
+  request('post', url, job).then((response: any) => {
+    if(response.status === 200) {
+      console.log(response.data)
+      successCallback && successCallback(response.data)
+    }
+  }).catch((error: any) => {
+    if(error.response)
+    {
+      failureCallback && failureCallback(error.response.data.message)
+      console.log(error.response)
+      notify(`Error: ${error.response.data.message}`, 'error');
+    }
+    console.log(error)
+  })
+}
```

Comment:
suggestion: this should be refactored as it was done for `ListOfficialImages` and `ListClusters`. Main points to be addressed:

- SubmitJobScript should return a Promise that resolves with the actual data. This is needed so that we can then use `react-query` for data fetching
- instead of using callbacks, we can subscribe to the returned promise and simply call `.then` and `.catch` if needed